addon/components/course/objective-list.hbs

```diff
@@ -1,23 +1,27 @@
 <div
   class="course-objective-list"
-  {{did-insert (perform this.load) (await @course.objectives)}}
-  {{did-update (perform this.load) (await @course.objectives)}}
   data-test-course-objective-list
+  {{did-insert (perform this.load) @course @course.objectives}}
```

Comment:
the `load()` task receives a second parameter with `@course.objectives` here, but it is not being processed. kruft or deliberate decision to pass this?

Reply:
It isn't used in `load` but load should be called again when it changed. It probably only needs to go on `did-update` to trigger the change, but I left it in both places because otherwise it looks like it is missing from one.